src/components/pager/pager.test.js

```diff
@@ -255,6 +256,44 @@ context("Test for Pager component", () => {
       firstArrow().should("be.visible");
       previousArrow().should("be.visible");
     });
+
+    it.each([
+      [false, "not.exist"],
+      [true, "be.visible"],
+    ])(
+      "when interactivePageNumber is %s, standard pager nav number input rendered correctly",
```

Comment:
```suggestion
      "when interactivePageNumber is %s, standard pager nav number input is rendered correctly",
```
This is literally the pettiest comment ever so I understand if you don't want to change it 😆